one/src/events.rs

```diff
@@ -28,9 +35,25 @@ pub struct SlurpOpts {
     output_ceramic_path: Option<PathBuf>,
 }
 
+#[derive(Args, Debug)]
+pub struct ValidateOpts {
+    /// Path to storage directory
+    #[arg(short, long, env = "CERAMIC_ONE_STORE_DIR")]
+    store_dir: Option<PathBuf>,
+
+    /// CID of the block to validate
+    #[arg(short, long)]
+    cid: Cid,
+
+    /// Ethereum RPC URL, e.g. ETHEREUM_RPC_URL=https://mainnet.infura.io/v3/<api_key>
+    #[arg(short, long, env = "ETHEREUM_RPC_URL")]
+    ethereum_rpc_url: String,
```

Comment:
this struct seems to represent the args for a single request to validate a single TimeEvent.  Copying around the eth_rpc_url for every single event we want to validate seems unnecessary?  Should we either store the eth_rpc_url as a member variable of a validator instance that is used for all validation requests, or take this by reference so we're not copying it for each validation request?

Reply:
this seems fine for now when it's only being used in the CLI command which will just validate a single event and then shut down.  But once we start validating every incoming time event on the standard codepaths, I don't think we're going to want to copy the rpc url for every single time event we need to validate.